Rubberduck.Refactorings/RemoveParameters/RemoveParametersModel.cs

```diff
@@ -79,10 +76,13 @@ private Declaration PromptIfTargetImplementsInterface()
             var message = string.Format(RubberduckUI.Refactoring_TargetIsInterfaceMemberImplementation,
                 declaration.IdentifierName, interfaceMember.ComponentName, interfaceMember.IdentifierName);
 
-            var confirm = _messageBox.ConfirmYesNo(message, RubberduckUI.ReorderParamsDialog_TitleText);
-            return confirm ? interfaceMember : null;
+            var args = new RefactoringConfirmEventArgs(message) {Confirm = true};
```

Comment:
The initial state of `Confirm` is always `true` when this is newed up? Why not just set it in the ctor?

Reply:
That's a good question. Will make a overload? optional argument? for the ctor to set the `Message`.

Reply:
AFAICT it's always being set to `true` when it's newed up.  Can't it just be initialized as `true`?  The other option would be to flip the conditional and just let it default to `false` and call it `Unconfirmed`, but I'm not sure I like those semantics as much.